lib/plugins/digging.js

```diff
@@ -1,6 +1,8 @@
 const nbt = require('prismarine-nbt')
 const { performance } = require('perf_hooks')
 const { createDoneTask, createTask } = require('../promise_utils')
+const { RaycastIterator, BlockFace } = require('prismarine-world').iterators
```

Comment:
once https://github.com/PrismarineJS/prismarine-world/pull/58 is merged you can use `bot.world.raycast()`

Reply:
Does this new raycasting still return every block within the raycasting path?

Reply:
it returns the end block but test every block shapes